src/shared/components/palette/organisation/write-organisation-field.component.html

```diff
@@ -54,7 +54,7 @@ <h2 class="heading-h2">Organisation name and address</h2>
           </div>
         </div>
       </div>
-      <div *ngIf="(selectedOrg$ | async).organisationIdentifier">
+      <div *ngIf="(selectedOrg$ | async)?.organisationIdentifier">
         <table id="organisation-selected-table" *ngIf="(selectedOrg$ | async) as selectedOrg">
```

Comment:
Again, this is covered by the `*ngIf` in the line directly above.

Reply:
this is diferent level